Changes

```diff
@@ -252,6 +252,7 @@ Working version
 ### Internal/compiler-libs changes:
 
 * GPR#1952: refactor the code responsible for displaying errors and warnings
+  `Location.report_error` is removed, use `Location.print_error` instead
```

Comment:
It's rather `Location.print_report` right?

Reply:
Indeed, thanks.